client/settings/connect-stripe-account/index.js

```diff
@@ -74,80 +63,37 @@ const ConnectStripeAccount = ( { oauthUrl, testOauthUrl } ) => {
-									'By clicking "Create or connect an account", you agree to the {{tosLink}}Terms of service.{{/tosLink}}',
-									'woocommerce-gateway-stripe'
-								),
-								components: {
-									tosLink: (
-										// eslint-disable-next-line jsx-a11y/anchor-has-content
-										<a
-											target="_blank"
-											rel="noreferrer"
-											href="https://wordpress.com/tos"
-										/>
-									),
-								},
-							} ) }
-						</TermsOfServiceText>
-						<p className="woocommerce-stripe-auth__help">
-							{ __(
-								'Some payment methods are automatically enabled when you connect your account. Review your Payment Methods settings for details.',
-								'woocommerce-gateway-stripe'
-							) }
-						</p>
-						<ButtonWrapper>
-							{ oauthUrl && (
-								<Button
-									variant="primary"
-									onClick={ handleCreateOrConnectAccount }
-								>
-									{ __(
-										'Create or connect an account',
-										'woocommerce-gateway-stripe'
-									) }
-								</Button>
-							) }
-							{ testOauthUrl && (
-								<Button
-									variant={
-										oauthUrl ? 'secondary' : 'primary'
-									}
-									onClick={ handleCreateOrConnectTestAccount }
-								>
-									{ oauthUrl
-										? __(
-												'Create or connect a test account instead',
-												'woocommerce-gateway-stripe'
-										  )
-										: __(
-												'Create or connect a test account',
-												'woocommerce-gateway-stripe'
-										  ) }
-								</Button>
-							) }
-						</ButtonWrapper>
-					</>
-				) : (
-					<InlineNotice isDismissible={ false } status="error">
-						{ interpolateComponents( {
-							mixedString: __(
-								'An issue occurred generating a connection to Stripe. Please try again. For more assistance, refer to our {{Link}}documentation{{/Link}}.',
-								'woocommerce-gateway-stripe'
+				<TermsOfServiceText>
+					{ interpolateComponents( {
+						mixedString: __(
+							'By clicking "Create or connect an account", you agree to the {{tosLink}}Terms of service.{{/tosLink}}',
+							'woocommerce-gateway-stripe'
+						),
+						components: {
+							tosLink: (
+								// eslint-disable-next-line jsx-a11y/anchor-has-content
+								<a
+									target="_blank"
+									rel="noreferrer"
+									href="https://wordpress.com/tos"
+								/>
 							),
-							components: {
-								Link: (
-									<ExternalLink href="https://woocommerce.com/document/stripe/setup-and-configuration/connecting-to-stripe/" />
-								),
-							},
-						} ) }
-					</InlineNotice>
-				) }
+						},
+					} ) }
+				</TermsOfServiceText>
+				<p className="woocommerce-stripe-auth__help">
+					{ __(
+						'Some payment methods are automatically enabled when you connect your account. Review your Payment Methods settings for details.',
+						'woocommerce-gateway-stripe'
+					) }
+				</p>
+				<ButtonWrapper>
+					<ConnectButton testMode={ false } buttonVariant="primary" />
```

Comment:
A subtle side-effect of this change is that we will _always_ show the option to connect a live/production account, regardless of the environment we're in, e.g. non-HTTPS and/or local URLs.
Can we replicate the original logic for this? It doesn't feel like a good idea to allow that.